src/app/api/collections/utils.py

```diff
@@ -10,7 +10,7 @@
 
 
 def map_elastic_response_to_model(
-    response: Response, specs: dict, model: Generic[T]
+    response: Response, specs: dict, model: Type[T]
```

Comment:
Maybe we can completely remove it and inline all its usage?

Reply:
As described in the tree.py comment, I prefer to have it here, because this way we have one central place for glom, not several places. This is more hexagonal-architecture like ;)

Reply:
reverted!